src/bot/modules/votes.ts

```diff
@@ -0,0 +1,159 @@
+import { correctionsChannelId, downReactionIdentifier, suggestionsChannelId, upReactionIdentifier } from '../constants';
+import { isGodfather } from '../utils';
+import {
+  Client,
+  GuildTextBasedChannel,
+  Message,
+  MessageReaction,
+  PartialMessageReaction,
+  PartialUser,
+  User
+} from 'discord.js';
+import prisma from '../../prisma';
+import { VoteType } from '@prisma/client';
+
+const Reactions = {
+  [upReactionIdentifier]: VoteType.UP,
+  [downReactionIdentifier]: VoteType.DOWN
+};
+
+export default class Votes {
+  public client: Client;
+
+  constructor(client: Client) {
+    this.client = client;
+  }
+
+  async run(partialReaction: MessageReaction | PartialMessageReaction, partialUser: User | PartialUser) {
+    if (!partialReaction.message.guild) return;
+    const reaction = await this.checkReaction(partialReaction);
+    if (!reaction) return;
+
+    const user = await this.checkUser(partialUser);
+    if (!user) return;
+
+    const channel = reaction.message.channel as GuildTextBasedChannel;
```

Comment:
Pourquoi pas définir la variable message juste au dessus pour que on puisse définir channel de cette façon ?
```suggestion
    const channel = message.channel as GuildTextBasedChannel;
```

Reply:
Je pense que cela est inutile, le code est assez lisible comme ça

Reply:
Je vois pas non plus l'intérêt.
Le path d'accès n'est pas suffisamment long pour que ça soit intéressant.